src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -510,7 +510,7 @@ function removeBlacklist(args, res) { // eslint-disable-line no-unused-vars
 
     const termIds = termFilters.map(termFilter => termFilter.id);
     
```

Comment:
I couldn't help notice that we're performing a combination of update and insert statements for cassandra records which I dont fully understand. when you invoke an insert statement in cassandra existing records are automatically updated.

Reply:
When I do: 
`INSERT INTO trustedsources (pipelinekey,externalsourceid,sourcetype,rank,insertiontime) VALUES ('Twitter','MonteriaRadio38','user',100,1505313678830);` 
and then 
`INSERT INTO trustedsources (pipelinekey,externalsourceid,sourcetype,rank,insertiontime) VALUES ('twitter','MonteriaRadio38','user',100,1505313678830);` 
I get 2 records.